### PR TITLE
[TE1.1] Support attributes read API (TimeFormatLocalization, UnitLocalization)

### PR DESCRIPTION
**Pull request by joonhaengHeo**
During Test Event client test cases, some API can't support android platform.
In this reason, I think we need to update the ZAP file to add API.

Time Format Localization cluser [TC-LTIME-1.1] Step 4. (attributeList)
https://github.com/CHIP-Specifications/chip-test-plans/blob/master/src/cluster/timeformatlocalization.adoc#test-procedure

Unit Localization Cluster [TC-LUNIT-1.1] Step 6, 7 (generatedCommandList, acceptedCommandList)
https://github.com/CHIP-Specifications/chip-test-plans/blob/master/src/cluster/unitlocalization.adoc#test-procedure

### CODE REVIEW

File: src/controller/data_model/controller-clusters.zap [hunk 1]
```diff
@@ -3588,7 +3588,7 @@
               "mfgCode": null,
               "side": "server",
               "type": "array",
-              "included": 0,
+              "included": 1,
```
Comment (bzbarsky-apple) [inline src/controller/data_model/controller-clusters.zap:3591]: Why is this needed?  Codegen should be ignoring these values for the global attributes which are always required to be supported.  If anything in codegen depends on getting these values right in the .zap, that thing is buggy and should be fixed.

Comment (bzbarsky-apple) [reply src/controller/data_model/controller-clusters.zap:3591]: And to be clear, making this change in the interests of expediency is probably OK, but we should fix the java codegen here to not be broken.

Comment (andy31415) [reply src/controller/data_model/controller-clusters.zap:3591]: I was not aware that global attributes are forced in codegen. I think matter idl generator should be updated in this case to make it clear what codegen thinks should exist.

Could you create an issue for this and assign to me? Maybe let me know which global attributes are hard coded.

In the mean time, I think this probably falls under "this is the faster path to get things working".

Comment (joonhaengHeo) [reply src/controller/data_model/controller-clusters.zap:3591]: @andy31415 @bzbarsky-apple 
I check https://github.com/project-chip/connectedhomeip/pull/24951.
The part that changes to this seems to be added only to the code file that is generated during the build. If so, it seems that additional modification of the ZAP file is needed to use in java, is it correct?

Comment (bzbarsky-apple) [reply src/controller/data_model/controller-clusters.zap:3591]: >  If so, it seems that additional modification of the ZAP file is needed to use in java, is it correct?

If that's correct, that's a bug in the java codegen that needs to be tracked and fixed.  In general, things are not really enabled much in controller-clusters nowadays, because all the non-java consumers of it, as far as I know, ignore what's enabled and what's disabled...

Comment (joonhaengHeo) [reply src/controller/data_model/controller-clusters.zap:3591]: I checked that this issue is merged in #24591. I will close this PR.
